src/components/common/date-input/DateInput.tsx

```diff
@@ -26,17 +33,22 @@ const DateInput = ({ id, label, value, onChange, inputWidth }: Props) => (
         id={id}
         autoOk
         disableToolbar
-        variant="inline"
+        disableFuture
+        variant="dialog"
         inputVariant="outlined"
         format="MM/DD/yyyy"
-        value={value}
-        onChange={(date) => onChange(moment(date).toDate())}
+        value={value === "" ? null : value}
+        onChange={(date) =>
+          onChange(
+            moment(date).isValid() ? moment(date).format("YYYY-MM-DD") : ""
```

Comment:
it seems like by doing this, users who enter invalid dates aren't seeing error messages anymore – is there any way we can keep those?

currently in main:
<img width="1499" alt="Screen Shot 2021-07-15 at 11 40 49 PM" src="https://user-images.githubusercontent.com/37346399/125888330-4a4cfaae-17a1-49e7-9c5c-f759fcc61ebd.png">

new:
<img width="1450" alt="Screen Shot 2021-07-15 at 11 40 12 PM" src="https://user-images.githubusercontent.com/37346399/125888279-79a44830-4aa3-4177-b20c-d82fb6c2dcd7.png">

Reply:
we can try changing the format to a string in `Field` if that's easier 🥴

Reply:
oh yikes my bad i'll look into this over the weekend and see if i can fix it 👌

Reply:
FIXED 🥳